src/Nancy/DefaultRootPathProvider.cs

```diff
@@ -14,7 +14,7 @@ public class DefaultRootPathProvider : IRootPathProvider
         public string GetRootPath()
         {
 #if CORE
-            return Microsoft.Extensions.PlatformAbstractions.PlatformServices.Default.Application.ApplicationBasePath;
+            return System.IO.Directory.GetCurrentDirectory();

```

Comment:
But we need a strategy because the status quo is deviating from happy path , having to implement a roothpath provider all the time.